### PR TITLE
[DE] add HassMediaPrevious

### PR DESCRIPTION
**Pull request by andreasbrett**
## Summary by CodeRabbit

- **New Features**
  - Introduced German language support for media player commands, including "next" and "previous/last" actions.

- **Enhancements**
  - Improved clarity and consistency of expressions for the "HassMediaNext" intent in German.

- **Tests**
  - Added comprehensive German test cases for the `HassMediaPrevious` intent, covering various devices and specific areas like the living room.

### CODE REVIEW

File: sentences/de/media_player_HassMediaPrevious.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+language: de
+intents:
+  HassMediaPrevious:
+    data:
+      - sentences:
+          - "[<starte> ](vorherige|letzte)[r|s|n][ <song>][ (<an>|am)][ artikel_bestimmt] <name>"
```
Comment (mib1185) [inline sentences/de/media_player_HassMediaPrevious.yaml:6]: we should make `(vorherige|letzte)[r|s|n]` an expansion rule

File: responses/de/HassMediaPrevious.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+language: de
+responses:
+  intents:
+    HassMediaPrevious:
+      default: "Vorheriges"
```
Comment (mib1185) [inline responses/de/HassMediaPrevious.yaml:5]:
```suggestion
      default: "Spiele vorheriges"
```

File: tests/de/media_player_HassMediaPrevious.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      - "Starte letzten Titel am TV"
+      - "Starte letzten Song am TV"
+      - "Spiele letztes Lied am TV"
+      - "Spiele letztes Stück am TV"
+      - "Letzter Titel auf dem TV"
+    intent:
+      name: HassMediaPrevious
+      slots:
+        name: "TV"
+    response: "Vorheriges"
```
Comment (mib1185) [inline tests/de/media_player_HassMediaPrevious.yaml:22]:
```suggestion
    response: "Spiele vorheriges"
```

File: tests/de/media_player_HassMediaPrevious.yaml [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+      - "Starte letzten Song im Wohnzimmer"
+      - "Spiele letztes Lied im Wohnzimmer"
+      - "Spiele letztes Stück im Wohnzimmer"
+    intent:
+      name: HassMediaPrevious
+      slots:
+        area: "Wohnzimmer"
+      context:
+        area: Wohnzimmer
+    response: "Vorheriges"
```
Comment (mib1185) [inline tests/de/media_player_HassMediaPrevious.yaml:40]:
```suggestion
    response: "Spiele vorheriges"
```

File: tests/de/media_player_HassMediaPrevious.yaml [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+      - "Starte letzten Song"
+      - "Spiele letztes Lied"
+      - "Spiele letztes Stück"
+    intent:
+      name: HassMediaPrevious
+      slots:
+        area: "Wohnzimmer"
+      context:
+        area: Wohnzimmer
+    response: "Vorheriges"
```
Comment (mib1185) [inline tests/de/media_player_HassMediaPrevious.yaml:58]:
```suggestion
    response: "Spiele vorheriges"
```

**Review by mib1185 — CHANGES_REQUESTED** [review]
Hi @andreasbrett 
many thanks for you contribution :slightly_smiling_face: 
I've left just some small comments

Further (_in a follow up PR_) we could also add sentences like "wiederhole den letzten Titel" or "zurück zum letzten Titel" (_like we have in the [English sentences](https://github.com/home-assistant/intents/blob/main/sentences/en/media_player_HassMediaPrevious.yaml)_)

File: sentences/de/media_player_HassMediaPrevious.yaml (hunk 1 above)
Comment (andreasbrett) [reply sentences/de/media_player_HassMediaPrevious.yaml]: implemented

**Review by mib1185 — APPROVED** [review]
Hi @andreasbrett , also thanks for this :+1:

**Pull request merged**